test/e2e/storage/ubernetes_lite_volumes.go

```diff
@@ -82,7 +83,7 @@ func PodsUseStaticPVsOrFail(ctx context.Context, f *framework.Framework, podCoun
 
 	zones, err := e2enode.GetSchedulableClusterZones(ctx, c)
 	framework.ExpectNoError(err)
-	zonelist := zones.List()
+	zonelist := sets.List(zones)
```

Comment:
Can you remove this for it will require the approval from sig storage, let's focus on sig scheduling only in the PR. Thanks and plz squash the commit. I'm LGTMed.

Reply:
@kerthcet there is a dependency on a file form where this file and files in scheduling are using the same function. They are required to go together. If I were to revert change on this file. The same revert would be required on some of the sig/scheduling files.

Reply:
@kerthcet I read in docs that reviewer can add squash label and then bot can automatically squash the commits. Can you help with that.

Reply:
Squash will combine the commit messages, better to squash by hand and remove the messages with no real value.

Reply:
And seems everyone can add the label https://github.com/kubernetes/test-infra/blob/3364943fd7cd305a1054428efc237b9fd42201ea/label_sync/labels.md#tide/merge-method-squash